tests/end-to-end/director/fields.go

```diff
@@ -159,10 +161,23 @@ func (fp *gqlFieldsProvider) ForLabel() string {
 }
 
 func (fp *gqlFieldsProvider) ForRuntime() string {
-	return fmt.Sprintf(`id
+	return fmt.Sprintf(`
+		id
 		name
 		description
 		labels 
 		status {condition timestamp}
 		agentAuth {%s}`, fp.ForAuth())
 }
+
+func (fp *gqlFieldsProvider) ForApplicationPage() interface{} {
```

Comment:
No need for this function, use `tc.gqlFieldsProvider.Page(tc.gqlFieldsProvider.ForApplication())` instead.